mm2src/mm2_main/tests/docker_tests/docker_tests_inner.rs

```diff
@@ -5471,3 +5471,96 @@ fn test_approve_erc20() {
 
     block_on(mm.stop()).unwrap();
 }
+
+#[test]
+fn test_peer_time_sync_validation() {
+    const TIMEOFFSET_TOLERABLE: i64 = 19;
+    const TIMEOFFSET_TOO_BIG: i64 = 21;
+
+    let start_peers_with_time_offset = |offset: i64| -> (Json, Json) {
+        let (_ctx, _, bob_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN", 10.into());
+        let (_ctx, _, alice_priv_key) = generate_utxo_coin_with_random_privkey("MYCOIN1", 10.into());
+        let coins = json!([mycoin_conf(1000), mycoin1_conf(1000)]);
+        let mut mm_bob = block_on(MarketMakerIt::start_with_envs(
+            json!({
+                "gui": "nogui",
+                "netid": 9000,
+                "dht": "on",  // Enable DHT without delay.
+                "passphrase": format!("0x{}", hex::encode(bob_priv_key)),
+                "coins": coins,
+                "rpc_password": "pass",
+                "i_am_seed": true,
+            }),
+            "pass".to_string(),
+            None,
+            &[],
+        ))
+        .unwrap();
+        let (_bob_dump_log, _bob_dump_dashboard) = mm_dump(&mm_bob.log_path);
+        block_on(mm_bob.wait_for_log(22., |log| log.contains(">>>>>>>>> DEX stats "))).unwrap();
+
+        let mut mm_alice = block_on(MarketMakerIt::start_with_envs(
+            json!({
+                "gui": "nogui",
+                "netid": 9000,
+                "dht": "on",  // Enable DHT without delay.
+                "passphrase": format!("0x{}", hex::encode(alice_priv_key)),
+                "coins": coins,
+                "rpc_password": "pass",
+                "seednodes": vec![format!("{}", mm_bob.ip)],
+            }),
```

Comment:
You can use `Mm2TestConf::seednode` to generate a dummy config

Reply:
fixed in [5be3e5c](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2304/commits/5be3e5cf4a07f456c7c0bfd6575772dcc5b68b74) thanks